.changeset/message-args.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": minor
+---
+
+Added: message arguments to most of the rules that pass them to the built-in messages
```

Comment:
```suggestion
Added: `messageArgs` to <number of> rules
```

Let's be explicit once we know the number of rules the pull request touches.

Reply:
@jeddy3 Done! Reverted the changes from the deprecated rules, and updated the changeset with the explicit count (77), also added the mention of the `message` option to all the READMEs, as it was reported by the “custom message option” test, did it consistently with the rules that already had it — a paragraph present before the `## Options` header.